.github/workflows/spark_master_test.yaml

```diff
@@ -42,11 +42,52 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
           sudo apt install libedit-dev
-        if: steps.git-diff.outputs.diff
```

Comment:
isn't this line important?

it says: `do X if there is a git diff that we care about`.

we shouldn't just delete it

Reply:
yeah sorry this was unintentional